packages/backend/src/open_payments/wallet_address/model.ts

```diff
@@ -53,6 +62,9 @@ export class WalletAddress
   public readonly assetId!: string
   public asset!: Asset
 
+  public readonly tenantId!: string
+  public tenant!: Tenant
```

Comment:
Do we actually need the `tenant!: Tenant` property on the model? I'm not sure we'd end up using it

Reply:
Originally I thought we would need it, but since we have `authServer` and `resourceServer`, we are good. Will update.